.circleci/scripts/git-diff-develop.ts

```diff
@@ -61,7 +56,7 @@ async function fetchUntilMergeBaseFound() {
     } catch (error: unknown) {
       if (
         error instanceof Error &&
-        hasProperty(error, 'code') &&
+        Object.hasOwnProperty.call(error, 'code') &&
```

Comment:
`hasProperty` is the only function being required from an external module. By inlining it, we can remove the dependency on `prep-deps` in CI.